portfolio/src/main/webapp/pages/scripts/admin.js

```diff
@@ -2,16 +2,54 @@ const main = () => {
 	addRows();
 };
 
-const addRows = async() => {
-	const response = await fetch("/getContacts");
-	const json = await response.text();
-	const contacts = JSON.parse(json);
+const addRows = async () => {
+	let response = await fetch("/getContacts");
+	let json = await response.text();
+	let contacts = JSON.parse(json);
+	// const contacts = [
+	// 	{
+	// 		name: "Test",
```

Comment:
Remove the commented code if not necessary. If you want it for a reference, mention a comment saying that the expected json structure.